iredis/client.py

```diff
@@ -81,6 +81,10 @@ def __init__(
         if prompt:
             self.prompt = prompt
 
+        self.verify_ssl = None
+        if self.scheme == "rediss":
+            self.verify_ssl = verify_ssl if verify_ssl is not None else "required"
+
```

Comment:
```suggestion
        self.verify_ssl = verify_ssl or "required"

```

Just a suggestion, I think `or` is more clear than `if else`

```
>>> None or "required"
'required'
>>> 
```

and since we have checked `scheme == "rediss"` in `create_connection`(checked when using), so I think can just skip the check here, always set the correct flag in client based on user's input.